### ISSUE TITLE
Page is too wide for browser

### ISSUE THREAD

**Issue by albertcui** [odota/web#1609]
**Current Behavior**
Match content overflows side of screen
![image](https://user-images.githubusercontent.com/3838552/40892749-a9789e76-674f-11e8-8b72-ed467fe09dbd.png)

**Environment**
ChromeOS Version 66.0.3359.158 (Official Build) (64-bit)

**Possible Solution**

Min width is set as 1200px which seems to be too large.

**Comment by howardchung** [comment]
https://www.opendota.com/matches/3931500559/vision

Note: at 533px the page transitions to mobile layout, maybe we need to do that earlier (at 1200px?)

**Comment by howardchung** [comment]
Actually does this affect all tabs?  What is your screen resolution?

**Comment by albertcui** [comment]
Yes, this affects all tabs. You can see the match header is messed up.

Resolution is 1200 x 800.

I think the problem is that we have a min width at all?

**Comment by howardchung** [comment]
Could try removing it.  I don't remember why it was added originally.  That said, I think the content will not fit in less than 1200 pixels, so it might not do anything.

**Comment by HenriqueFadoni** [comment]
It can also be a bug related to the size of the div with the min-div. I'll try to fix it, thanks for the info guys!

**Comment by HenriqueFadoni** [comment]
I've found the bug, and I fixed it. However, I cannot find the divs to apply the css

**Comment by HenriqueFadoni** [comment]
![image](https://user-images.githubusercontent.com/38730725/51806806-a19d6980-2265-11e9-9355-089f3227eb42.png)

**Comment by HenriqueFadoni** [comment]
Never mind, I fixed it, I'll do the PR

**Comment by HenriqueFadoni** [comment]
Can someone approve my pull request? It already passed the tests, I just need a reviewer with write access.
